mariadb/charts/persistence/templates/_helpers.tpl

```diff
@@ -0,0 +1,18 @@
+{{- define "persistence.volume" -}}
+      {{- if .Values.persistence.enabled -}}
```

Comment:
This is actually tricky because the template is called in the parent chart (https://github.com/viglesiasce/charts/pull/1/files#diff-7c40a3a3972d5cbf2448cc743ed7a320R60), and that's where the context of `.` gets passed. It's confusing though, can you think of another way to do this?